specs/pvs/corla.pvs

```diff
@@ -372,8 +405,8 @@ BEGIN
 % @todo Each scanner has a unique id number.
   vvpat: TYPE
```

Comment:
We don't know.  Given recent comments from CDOS, it sounds like they have some local terminology we need to learn about VV* in general.  @nealmcb, can you give us a brief tutorial on such on Slack?